Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -958,6 +1127,58 @@ def route_edl() -> Response:
     return resp
 
 
+@APP.route('/log', methods=['GET'])
+def route_edl_log() -> Response:
+    """Show the edl indicators log on '/log' API request. """
+    params = demisto.params()
+
+    cache_refresh_rate: str = params.get('cache_refresh_rate')
+    auth_resp = authenticate_app(params, request.headers)
+    if auth_resp:
+        return auth_resp
+
+    created = datetime.now(timezone.utc)
+    request_args = get_request_args(request.args, params)
+    demisto.debug("Getting log file to show")
+    edl_data_log = ''
+    if os.path.exists(EDL_FULL_LOG_PATH):
+        demisto.debug("found log file")
+        with open(EDL_FULL_LOG_PATH, 'r') as log_file:
+            log_file.seek(0)
+            edl_data_log = log_file.read()
+            log_file.seek(0)
+
+    if len(edl_data_log) == 0:
+        edl_data_log = '# Empty'
```

Comment:
Let's extract this to a separate function
```suggestion
    def get_edl_log_file():
        if os.path.exists(EDL_FULL_LOG_PATH):
            demisto.debug("found log file")
            with open(EDL_FULL_LOG_PATH, 'r') as log_file:
                log_file.seek(0)
                edl_data_log = log_file.read()
                log_file.seek(0)

    
    edl_data_log = get_edl_log_file() or '# Empty'
        
```